.github/workflows/deploy_production.yaml

```diff
@@ -16,6 +16,17 @@ jobs:
       PULL_USER: ${{ secrets.PULL_USER }}
       PULL_PERSONAL_ACCESS_KEY: ${{ secrets.PULL_PERSONAL_ACCESS_KEY }}
     steps:
+      - name: Check tag
+        id: 'check-tag'
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs\/tags\/release-v[0-9]+\.[0-9]+\.[0-9]+$ ]]
```

Comment:
Could you please write comment about reason of this check
Presence of a wildcard RegExp in `on` node above confuses in case when out of discussion context, because a related task contains no information.

Reply:
There are two main reasons of this change. First one is restriction of action running with branch as an input source. And second reason is fix tag format with provided regular expression